src/Modules/CalcSections.lua

```diff
@@ -345,6 +345,9 @@ return {
 		{ label = "DPS Multiplier", modName = "DPS" }, 
 	}, },
 	{ label = "Skill DPS", flag = "triggered", { format = "{1:output:TotalDPS}", { breakdown = "TotalDPS" }, { label = "DPS Multiplier", modName = "DPS" }, }, },
+	{ label = "Max Effective APS", haveOutput = "KineticFusilladeMaxEffectiveAttackRate", { format = "{2:output:KineticFusilladeMaxEffectiveAttackRate}", }, },
```

Comment:
Same with showing the stats in multiple boxes here. If you have to show a breakdown for the duration and constant attacking variant you can add that a it's own breakdown under the `Skill type-specific stat` section as we've done for ball lightning